### PR TITLE
Owen Kelly homework 5

### PR DESCRIPTION
**Pull request by owenkelly**

### CODE REVIEW

File: routes/truckRoutes.js [hunk 1]
```diff
@@ -1 +1,57 @@
...
+router.route('/trucks/:name')
+  .get(function(req, res){
+    var truck = req.params.name;
+    var foodtruck = trucks.getTruck(truck);
+    res.send(foodtruck);
+  })
+  .delete(function(req, res){
+    var truck = req.params.name;
+    trucks.removeTruck(truck);
+    res.status(200);
```
Comment (jamdon) [inline routes/truckRoutes.js]: You'e set the status here, but you never send any response, which means the server call just hangs (DELETE request from the browser goes into a pending state and, depending on the browser, will eventually time out.)

**Comment by jamdon** [comment]
I've left an inline comment about an issue in your DELETE route.

**Comment by owenkelly** [comment]
I believe I've fixed the issue.

File: routes/truckRoutes.js [hunk 2]
```diff
@@ -1 +1,57 @@
-// put your Express router code in here
\ No newline at end of file
+var express = require('express');
+var bodyParser = require('body-parser');
+var urlEncoded = bodyParser.urlencoded({ extend: false });
```
Comment (jamdon) [inline routes/truckRoutes.js:3]: It's `extended` instead of `extend` I made this mistake myself before.

**Comment by jamdon** [comment]
Works good. Left one additional inline comment. 👍

**Pull request closed without merging**